assets/templates/datasetLandingPage/filterable.tmpl

```diff
@@ -159,13 +159,13 @@
             {{if .DatasetLandingPage.QMIURL}}
             <li>
              <div id="id-qmi" class="section__content--markdown margin-right--0">
-               <h2 class="font-weight-700 padding-top--0 padding-left-sm--2 padding-left-md--3">Quality and methodology information</h2>
+               <h2 class="font-size--32 line-height--40 font-weight-700 padding-top--0 padding-left-sm--2 padding-left-md--3">Quality and methodology information</h2>
                <p class="margin-bottom--0">Includes:</p>
                <ul>
-                 <li>Where the data comes from (source)</li>
-                 <li>The accuracy and reliability of the data</li>
-                 <li>The relevance of the data to certain applications or uses</li>
-                 <li>Things to consider when using the data</li>
+                 <li class="font-size--18 line-height--32">Where the data comes from (source)</li>
```

Comment:
Should be fine to remove these `font-size--18` classes here too because of `adjust-font-size` being set above